website/docs/registry/policy-libraries/publishing.mdx

```diff
@@ -0,0 +1,73 @@
+---
+page_title: Terraform Registry - Publishing Policy Libraries
+description: >-
+  How to publish Policy Libraries to the Terraform Registry.
+---
+
+# Publishing Policy Libraries
+
+A policy library is a collection of [Sentinel policies](https://docs.hashicorp.com/sentinel/writing) and [policy modules](https://docs.hashicorp.com/sentinel/extending/modules) that anyone can share through the Terraform Registry.
+
+Policy libraries are built and maintained on GitHub. Once a policy library has been published, you can release a new version of the policies by pushing a properly formatted Git tag.
+
+## Requirements for a Policy Library
+
+- **Public GitHub repository**: The files defining Sentinel policies and policy modules must be stored in a public GitHub repository.
+
+- **Repository name**: GitHub repositories containing Policy Libraries must be named according to the convention `policy-library-<NAME>`, where the `<NAME>` segment provides a name for your policy library.
+
+- **Repository description**: The repository's description is displayed in search and browse pages of the Registry. This should be a short, one sentence description of purpose of the policy library.
+
+- **sentinel.hcl file**: The GitHub repository must include a `sentinel.hcl` file at its root level. This file outlines the names and file locations for policies and policy modules that are included in the policy library. This is a [Sentinel configuration file](https://docs.hashicorp.com/sentinel/configuration), and it uses the same syntax that a Sentinel author is already familiar with. There are two caveats:
```

Comment:
Should we mention that we will ignore all the policies that aren't declared on the `sentinel.hcl` file?